stable/mongodb/values-production.yaml

```diff
@@ -138,6 +138,8 @@ resources: {}
 #   cpu: 100m
 #   memory: 256Mi
 
+# priorityClassName: ""
```

Comment:
Can you add a comment before the parameter? Something like:

```
## Pod priority
## https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
# priorityClassName: ""
```

Reply:
done!